lighthouse-core/audits/estimated-input-latency.js

```diff
@@ -79,17 +77,28 @@ class EstimatedInputLatency extends Audit {
    * @return {!Promise<!AuditResult>} The score from the audit, ranging from 0-100.
    */
   static audit(artifacts) {
-    const trace = artifacts.traces[this.DEFAULT_PASS];
+    return new Promise((resolve, reject) => {
```

Comment:
is this wrapping promise necessary? should be able to just return `computedP.catch(generateError)`?